Back-end/src/main/java/com/bramgussekloo/projects/services/UserService.java

```diff
@@ -0,0 +1,61 @@
+package com.bramgussekloo.projects.services;
+
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+
+import com.bramgussekloo.projects.config.SecurityConfig;
+import com.bramgussekloo.projects.exceptions.BadRequestException;
+import com.bramgussekloo.projects.exceptions.NotFoundException;
+import com.bramgussekloo.projects.models.User;
+import com.bramgussekloo.projects.repositories.UserRepository;
+
+@Service
+public class UserService {
+    
+    private final UserRepository repository;
+
+    public UserService(UserRepository repository){
+        this.repository = repository;
+    }
+
+    public Iterable<User> getAll() throws Exception {
+        return repository.findAll();
+    }
+
+    public User create(User user) throws Exception {
+        boolean userExists = false;
+        Iterable<User> users = getAll();
+
+        for (User userFromList : users) {
+            if (user.getUser_name().equals(userFromList.getUser_name())) {
+                userExists = true;
+                break;
+            }
+        }
+
+        if (!userExists) {
+            if (!user.getPassword().isEmpty() && user.getPassword().length() > 6) {
+                user.setPassword(SecurityConfig.HashUserPassword(user.getPassword()));
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.